### DISCUSSION TITLE
Transifex translation

### DISCUSSION THREAD

**Discussion by KrystianPlichta** [modoboa/modoboa#2435] [Help]
Polish translation of modoboa in transifex has reached 100% translation. After installing modoboa still many words are untranslated as if they were not downloaded. 
What can I do to get an up-to-date translation from transifex?

**Comment by comradekingu** [comment]
@KrystianPlichta I don't know, but you can build the project yourself. Or you can try to submit your translations as a PR on GitHub.
Download the language files from Transifex and keep them handy once the project hopefully moves to a different translation platform.
